packages/ui/src/DateTimeActionSheet.tsx

```diff
@@ -113,30 +124,26 @@ const CalendarHeader = ({
   const displayDate = DateTime.fromJSDate(month[0]).toFormat("MMM yyyy");
   return (
     <Box alignItems="center" direction="row" height={40} justifyContent="between" width="100%">
-      <IconButton
-        accessibilityLabel="Go back 1 year"
+      <CalendarButton
```

Comment:
Do we need to add an accessibilityLabel prop to this component?

Reply:
Ooo good call!